docs/release-notes-v22.1.1-collab.md

```diff
@@ -0,0 +1,201 @@
+* [#4301](https://github.com/redpanda-data/redpanda/issues/4301) New admin api request `/v1/debug/partition_leaders_table`. by @VadimPlh in https://github.com/redpanda-data/redpanda/pull/4259
+* [#4333](https://github.com/redpanda-data/redpanda/pull/4333) rpk: add broker version info in `redpanda admin broker ls`. by @r-vasquez in https://github.com/redpanda-data/redpanda/pull/4409
+* [#4402](https://github.com/redpanda-data/redpanda/issues/4402) operator: added hooks for rolling restarts and upgrades. by @nicolaferraro in https://github.com/redpanda-data/redpanda/pull/4385
+* operator: Allow Redpanda resources to be directly configured. by @BenPope in https://github.com/redpanda-data/redpanda/pull/3773
+* schema_registry: Support `GET /subjects/{subject}/versions/{version}/referencedBy`. by @BenPope in https://github.com/redpanda-data/redpanda/pull/3299
+* Add cluster level configuration parameters `cloud_storage_enable_remote_read` and `cloud_storage_enable_remote_write` that can be used to enable shadow indexing for all topics. by @Lazin in https://github.com/redpanda-data/redpanda/pull/3233
+* Add settings for `max_old_gen_size`. by @VadimPlh in https://github.com/redpanda-data/redpanda/pull/3167
+* Admin API requests sent to non-leader nodes are redirected more reliably to the leader. by @jcsp in https://github.com/redpanda-data/redpanda/pull/3571
+* Cluster status stanza will have version field for convenience. by @pvsune in https://github.com/redpanda-data/redpanda/pull/4207
+* Configuration property `kafka_connections_max_overrides` is added, enabling setting connection count limits on individual client IPs. by @jcsp in https://github.com/redpanda-data/redpanda/pull/4221
+* Kafka: return shadow indexing configs in a topic describe handler. by @LenaAn in https://github.com/redpanda-data/redpanda/pull/3192
+* Kubernetes operator: All Redpanda nodes created through the Kubernetes operator will have a default cloud storage maximum upload interval (`cloud_storage_segment_max_upload_interval_sec`) of 30 minutes. by @0x5d in https://github.com/redpanda-data/redpanda/pull/3218
+* Made `min_free_memory` parameter of background reclaimer configurable. by @mmaslankaprv in https://github.com/redpanda-data/redpanda/pull/3410
+* New admin API endpoint `PUT /v1/features/<feature>`, for use on future feature flags which may not be enabled by default. by @jcsp in https://github.com/redpanda-data/redpanda/pull/3936
+* Redpanda now has an internal store for configuration properties and many properties can be set without restarting redpanda. When you upgrade Redpanda, your existing configuration will be imported automatically. Please consult the documentation for more detail on this feature. New rpk subcommands for handling cluster configuration: `rpk cluster config [edit|import|export|set|get|status|force-reset|lint]`. by @jcsp in https://github.com/redpanda-data/redpanda/pull/3760
```

Comment:
This is centralized config and should be up in the centralized config section

Reply:
nice catch thanks. fixed